### ISSUE TITLE
use pictures instead of camera

### ISSUE THREAD

**Issue by salsa2k** [nomadoda/react-zxing#5]
There is any way to use pictures instead of the camera?

Thanks

**Comment by adamkrupke** [comment]
It's a good idea. What do you have in mind? To upload an image and decode it?

**Comment by salsa2k** [comment]
yes, correct, some devices like Amazon Tablets don't have a camera barcode scan. Would be nice to be able to take a photo and use it to decode.

**Comment by atengberg** [comment]
On one of the other html5-qrcode libraries, they use file input as a fallback which seemed a good way to handle this (additionally as another callback for debugging this would also be helpful).

**Comment by adamjanicki2** [comment]
> It's a good idea. What do you have in mind? To upload an image and decode it?

An easier idea would be to allow passing a `ref` to an `img` element instead of only being able to pass it to a `video` element -- that way you wouldn't have to worry about implementing any sort of upload input/element
